modules/nf-core/ctree/main.nf

```diff
@@ -0,0 +1,56 @@
+process CTREE {
+    tag "$meta.id"
+    label "process_low"
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/r-ctree_r-mobster_r-viber_r-cli_pruned:660fb0bfa4d8d4ac':
+        'community.wave.seqera.io/library/r-ctree_r-mobster_r-viber_r-cli_pruned:48299db4104e296b' }"
+
+    input:
+        tuple val(meta), path(ctree_input)
+
+    output:
+        tuple val(meta), path("**ctree_{mobster,VIBER,pyclonevi}.rds")             , emit: ctree_rds       , optional: true
+        tuple val(meta), path("**ctree_{mobster,VIBER,pyclonevi}_plots.rds")       , emit: ctree_plots_rds , optional: true
+        tuple val(meta), path("**REPORT_plots_ctree_{mobster,VIBER,pyclonevi}.rds"), emit: ctree_report_rds, optional: true
+        tuple val(meta), path("**REPORT_plots_ctree_{mobster,VIBER,pyclonevi}.pdf"), emit: ctree_report_pdf, optional: true
+        tuple val(meta), path("**REPORT_plots_ctree_{mobster,VIBER,pyclonevi}.png"), emit: ctree_report_png, optional: true
+        path "versions.yml"                                                        , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    template "main_script.R"
+
+    stub:
```

Comment:
How does the tool know whether it is mobster or VIBER or whatever the third is?
Because the stub here is assuming all mobster?

Reply:
Why can you not just touch the output files that you'd see for VIBER?